app/controllers/application_controller.rb

```diff
@@ -2,4 +2,7 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  def after_sign_out_path_for(resource_or_scope)
```

Comment:
This function is been removed in next PRs, this function is redundant in current application for current use case